testsuite/features/secondary/proxy_cobbler_pxeboot.feature

```diff
@@ -0,0 +1,175 @@
+    When I follow "Details" in the content area
+    And I follow "Connection" in the content area
+    Then I should see "proxy" short hostname
+
+  Scenario: Install a package on the PXE boot minion
+    Given I am on the Systems overview page of this "pxeboot_minion"
+    When I install the GPG key of the test packages repository on the PXE boot minion
+    And I follow "Software" in the content area
+    And I follow "Install"
+    And I check "virgo-dummy-2.0-1.1" in the list
+    And I click on "Install Selected Packages"
+    And I click on "Confirm"
+    Then I should see a "1 package install has been scheduled" text
+    When I wait until event "Package Install/Upgrade scheduled by admin" is completed
+
+  Scenario: Cleanup: remove the auto installation profile
+    Given I follow the left menu "Systems > Autoinstallation > Profiles"
+    When I follow "15-sp2-cobbler"
+    And I follow "Delete Autoinstallation"
+    And I click on "Delete Autoinstallation"
+    Then I should not see a "15-sp2-cobbler" text
+
+  Scenario: Cleanup: remove the auto installation distribution
+    Given I follow the left menu "Systems > Autoinstallation > Distributions"
+    When I follow "SLE-15-SP2-TFTP"
+    And I follow "Delete Distribution"
+    And I click on "Delete Distribution"
+    And I remove package "tftpboot-installation-SLE-15-SP2-x86_64" from this "server"
+    And I wait for "tftpboot-installation-SLE-15-SP2-x86_64" to be uninstalled on "server"
+    Then I should not see a "SLE-15-SP2-TFTP" text
+
+  Scenario: Cleanup: delete the PXE boot minion
+    Given I am on the Systems overview page of this "pxeboot_minion"
+    When I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    Then "pxeboot_minion" should not be registered
+    And I stop salt-minion on the PXE boot minion
+
+  Scenario: Cleanup: remove DNS records added by mass import
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "Formulas" in the content area
+    And I follow first "Bind" in the content area
+    # direct zone tf.local:
+    And I scroll to the top of the page
+    And I press minus sign in third configured zone section
+    And I press minus sign in third available zone section
+    And I click on "Save Formula"
+    Then I should see a "Formula saved" text
+
+  Scenario: Cleanup: the PXE boot minion prefers booting via saltboot
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "Formulas" in the content area
+    And I follow first "Dhcpd" in the content area
+    And I enter "boot/pxelinux.0" in pxeboot filename field
+    And I click on "Save Formula"
+    Then I should see a "Formula saved" text
+
+  Scenario: Cleanup: apply the highstate after the formula cleanup changes
+    Given I am on the Systems overview page of this "proxy"
```

Comment:
Unnecessary line

Reply:
Too many of them! Cool, we save many steps!